src/operators/skip-some-property-nil.test.ts

```diff
@@ -1 +1,43 @@
-// TODO: implement tests
+import { map } from "rxjs/operators";
+import { assert } from "chai";
+import { of } from "rxjs";
+import { hasNoNilProperties, skipSomePropertyNil } from "./skip-some-property-nil";
+
+describe("hasNoNilProperties", function () {
+
+  it("works", async function () {
+    const obj = {
+      a: 1 as number | null,
+      b: 2 as number | undefined,
+    };
+
+    // obj.a.toExponential(); // check: compiler error
+    // obj.b.toExponential(); // check: compiler error
+
+    if (hasNoNilProperties(obj)) {
```

Comment:
test only tests compilation and has no expectation. but we also need a runtime test for the predicate function. ts compiler believes in our implementation.

ah, now I saw the functional test in the operator function test below. i would love to see it in the predicate test.